tests/schema_update_test.go

```diff
@@ -264,7 +278,7 @@ func TestVCSSchemaUpdate(t *testing.T) {
 		Name:               "Test Repository",
 		FullPath:           repositoryPath,
 		WebURL:             fmt.Sprintf("%s/%s", gitURL, repositoryPath),
-		BranchFilter:       "master",
+		BranchFilter:       "foo",
```

Comment:
if branch is feature/foo, branchFilter should corresponds to feature/*

https://docs.gitlab.com/ee/user/project/integrations/webhooks.html#filter-push-events-by-branch

https://gitlab.com/gitlab-org/gitlab-foss/-/issues/20338

Reply:
I see, done.